packages/shared/src/components/squads/Details.tsx

```diff
@@ -59,26 +68,56 @@ export function SquadDetails({
   onSubmit,
   form,
   createMode = true,
+  children,
 }: SquadDetailsProps): ReactElement {
+  const { user } = useAuthContext();
   const {
     name,
     handle,
     description,
     memberPostingRole: initialMemberPostingRole,
     memberInviteRole: initialMemberInviteRole,
   } = form;
+  const { data: status } = useQuery<SquadStatus>(
+    generateQueryKey(RequestKey.SquadStatus, user),
+    () => {
+      // TODO: MI-344 we need to add a condition if they are already an approved public squad that went private to return approved
```

Comment:
Wait, so we want a private squad that once went public but returned to private to say approved?

Have we confirmed this in any RFC? Sorry if yes just can't remember so just checking

Reply:
Not sure if it was in the RFC, but if you have gone through the approval process already, I don't think you should run though it again. Let me confirm with the product team.

Reply:
product thread: https://dailydotdev.slack.com/archives/C069XNYMDGV/p1715851745713089